test/e2e/tilesets-import.test.js

```diff
@@ -100,7 +97,10 @@ test('POST /tilesets/import creates tileset', async (t) => {
 
     const tileset = tilesetGetResponse.json()
 
-    t.same(tileset, createdTileset)
+    // TODO: Once we replace GET /tilesets/:id with a JS API, we shouldn't need
+    // this `undefined` removal hack.
+    // See <https://github.com/digidem/mapeo-map-server/issues/111>.
```

Comment:
I could do a better job here but figured it wasn't worth it, given that we're going to drop it (and it's test-only).